sdks/python/setup.py

```diff
@@ -69,7 +69,7 @@ def get_version():
   )
 
 
-REQUIRED_CYTHON_VERSION = '0.26.1'
+REQUIRED_CYTHON_VERSION = '0.28.1'
```

Comment:
<!--new_thread; commit:05e95b6995f0b88a53930c30459c8d81e6692219; resolved:0-->
How was this version bump determined?